src/components/tabs/index.tsx

```diff
@@ -7,6 +7,7 @@ import { Component, Element, Event, EventEmitter, h, Host, Listen, State, Watch
 })
 export class SmoothlyTabs {
 	@Element() element: HTMLSmoothlyTabsElement
+	@Prop({ reflect: true }) hideTabsIfSingle: boolean
```

Comment:
How about:
`show: "always" | "multiple" = "always"`

Reply:
always here is a bit confusing.